### ISSUE TITLE
Accessibility: Lacking support for keyboard tabbing focus rects

### ISSUE THREAD

**Issue by psyked** [internetarchive/openlibrary#1006]
It doesn't seem to be possible to visually identify which elements are focussed when navigating the page with a keyboard or similar assistive technology, presumably because CSS has been applied which disables the default focus rectangle of the browser. I don't believe that this should be disabled entirely, and it should either be restored to the browser default or an on-brand version of the focus reticle should exist.

**Comment by LeadSongDog** [comment]
This may also be seen when editing with Chrome on iOS. Tapping the edit box for some fields will often produce no visible change of focus.

**Comment by psyked** [comment]
My preference would be to tackle element and form field focus in the same manner as the gov.uk site does - with a clear, clean outline for elements on focus, which can be seen here: https://govuk-elements.herokuapp.com/form-elements/

**Comment by psyked** [comment]
A useful article to support the re-introduction of this browser default behaviour is here: https://www.tjvantoll.com/2013/01/28/stop-messing-with-the-browsers-default-focus-outline/
